docs/standard/analyzers/api-analyzer.md

```diff
@@ -0,0 +1,86 @@
+---
+title: .NET API analyzer
+description: Learn how the .NET API Analyzer can help detect deprecated APIs and platform compatibility issues.
+author: oliag
+ms.author: mairaw
+manager: 
+ms.date: 01/12/2018
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+---
+# .NET API analyzer
+
+The .NET API Analyzer is a Roslyn analyzer that discovers potential compatibility risks for C# APIs on different platforms and detects calls to deprecated APIs. It can be useful for all C# developers at any stage of development.
+
+API Analyzer comes as a NuGet package [Microsoft.DotNet.Analyzers.Compatibility](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/) and after you reference it in a project, it automatically monitors the code and indicates problematic API usage. You can also get suggestions on possible fixes by clicking on the light bulb. The drop-down menu includes an option to suppress the warnings.
+
+> [!NOTE]
+> The .NET API analyzer is still a pre-release version.
+
```

Comment:
I think we need to add a prerequisites section. Which version of VS we need and what kind of project. I tested this in 2 projects: a new .NET Core project and an existing .NET Framework 4.7. It only worked on the new .NET Core.